modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -1674,29 +1697,13 @@ private void _registerNamespace(
 		for (ServletData servletData : servletDatas) {
 			Set<String> graphQLNamespaces = new HashSet<>();
 
+			String defaultGraphQLNamespace = _getDefaultGraphQLNamespace(
```

Comment:
You are accessing inside this method the value of the `servletData.getPath`, but before, we were validating that is not null